### ISSUE TITLE
paper-item accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-item#9]
Contrast: the contrast on the toggle sliders is really low - it's very difficult to see the button to slide it. Also, is the gray focus state a lighter shade of gray on the toggle buttons than when it's on the checkboxes, or am I just seeing it differently? It appears lighter to my eyes, which makes it harder to see. 

Also, the contrast is poor in the second lines of text, e.g. where it says "Change your Google+ Profile Photo" under Profile Photo. Any chance we can make that text slightly darker? It's very difficult to see. 

I am not able to hear spoken feedback for the avatars or icons. If they don't contain useful information, that's fine. Will you be giving developers the ability to add labels to these if they would like though?

Accessibility Dev Tools audit results: 
[Warning] These elements are focusable but either invisible or obscured by another element (4)
Not applicable tests (7)
Passing tests (7)

**Comment by morethanreal** [comment]
> Also, the contrast is poor in the second lines of text, e.g. where it says "Change your Google+ Profile Photo" under Profile Photo. Any chance we can make that text slightly darker? It's very difficult to see.

I based the colors off the Material Design spec (54% black for secondary text). Can we chat with the design team if there are accessibility issues with the colors, so there is a place where the proper colors are documented?

> I am not able to hear spoken feedback for the avatars or icons. If they don't contain useful information, that's fine. Will you be giving developers the ability to add labels to these if they would like though?

Yes, the avatars and icons should be provided by the developer. Those elements which are provided by other repositories should implement `aria-label` if needed, or the developer may add a custom `aria-label` to the item if that is more apprpriate.

**Comment by lpalmaro** [comment]
+Hwi. Thanks! Responses in line.

On Fri, May 22, 2015 at 2:53 PM, Yvonne Yip <notifications@github.com>
wrote:

> Also, the contrast is poor in the second lines of text, e.g. where it says
> "Change your Google+ Profile Photo" under Profile Photo. Any chance we can
> make that text slightly darker? It's very difficult to see.
>
> I based the colors off the Material Design spec (54% black for secondary
> text). Can we chat with the design team if there are accessibility issues
> with the colors, so there is a place where the proper colors are documented?
>
> Hwi - what do you suggest we do here? Who should we chat with about this,
in Bethany's absence?

**Comment by lpalmaro** [comment]
Quick follow up -- Hwi and I just looked at the demo page again (which
looks a bit different than when I tested a few days ago?) and the contrast
is meeting our guidelines. We should be fine here. Thanks!

**Comment by morethanreal** [comment]
Closing.
